packages/features/filters/components/TeamsFilter.tsx

```diff
@@ -17,9 +17,10 @@ import { filterQuerySchema } from "../lib/getTeamsFiltersFromQuery";
 export type IEventTypesFilters = RouterOutputs["viewer"]["eventTypes"]["listWithTeam"];
 export type IEventTypeFilter = IEventTypesFilters[0];
 
+export const TEAMS_FILTER_KEY = "TEAMS_FILTER_KEY";
```

Comment:
To reduce possibility that "TEAMS_FILTER_KEY" is used elsewhere I would append "EVENT_TYPES_TEAMS_FILTER_KEY" in front to reduce possibility of collision but its your call.

Reply:
Sure will also change this

Reply:
Yeah actually now that i think about it the current implementation is good - i mistakenly thought the file was in event-types related folder.